### ISSUE TITLE
Malformed playlist field leads to confusing 'total' error

### ISSUE THREAD

**Issue by emallove** [plamere/SmarterPlaylists#31]
The playlist URI field must be in `spotify:playlist:<ID>` format, e.g., the below HTTP URL format will not work:

![spotify playlist -- in URL form](https://user-images.githubusercontent.com/711944/168442502-be7debe8-f1ea-4b6b-8e63-08b7bb7aa3f5.png)

But the error message caused by the malformed playlist field is confusing (red div with the text 'total'):

![confounding total red error message up top](https://user-images.githubusercontent.com/711944/168442501-1a51ec60-12b6-428e-96b9-fc1aa2724a27.png)

**Comment by Alyx611** [comment]
I had the same issue. I fixed it by changing the uri field from this fromat
"https://open.spotify.com/playlist/**PLAYLIST_ID**?si=123456789"
to this
"spotify:user:spotify:playlist:**PLAYLIST_ID**"

**Comment by flawedgun** [comment]
> I had the same issue. I fixed it by changing the uri field from this fromat "https://open.spotify.com/playlist/**PLAYLIST_ID**?si=123456789" to this "spotify:user:spotify:playlist:**PLAYLIST_ID**"

Agree; that's the way to format the URI. The issue @emallove is raising is that the error message is unhelpful for users trying to resolve the error (i.e., there's nothing to indicate that the URI was incorrectly formatted, the issue could be anything).

**Comment by larralapid** [comment]
Also just found this out after 10+ years of being a Spotify user: **When you right click on a playlist in the desktop app and navigate your cursor to the "share" menu item, if you have the option butten held down, instead of copying the URL to the playlist, you can instead copy the Spotify URI.**

**Comment by beacharoni** [comment]
Thanks :)

Am So., 29. Okt. 2023 um 23:58 Uhr schrieb larra ***@***.***>:
